### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Adarshvarghese**
testing a new branch for issue

### CODE REVIEW

**Comment by Adarshvarghese** [comment]
i have made the suggested changes .please have look and let me know if i have missed anything.

File: public/app.js
Comment (ajayesivan) [inline public/app.js:1]: Please remove the new lines added in this line since it is not required.

File: public/styles.css [hunk 1]
```diff
@@ -84,13 +87,33 @@
   background-color: transparent;
   border-color: transparent;
 }
+.header-bar{
+  max-width: 1300px;
```
Comment (ajayesivan) [inline public/styles.css:91]: Remove the `max-width` property from `header-bar`, we already have that in the content wrapper.

File: public/styles.css [hunk 2]
```diff
@@ -84,13 +87,33 @@
...
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  top: 0px;
+  /* left: 0px; */
```
Comment (ajayesivan) [inline public/styles.css:96]: We need `left` property, please add that back.

File: public/styles.css [hunk 3]
```diff
@@ -505,7 +538,11 @@ html {
 
   .menu {
     display: block;
-  }
+   }
+.nav-bar{
+  margin-right: 50px; 
+ }
```
Comment (ajayesivan) [inline public/styles.css:544]: Please remove this since we don't need a `margin-right` here.

File: public/styles.css [hunk 4]
```diff
@@ -399,6 +424,14 @@ html {
...
 
   }
+  .menu-option {
+    margin-bottom: 10px;
+    padding-bottom: 25px;
+  }
+  .menu-nav{
+    padding-top: 125px;
+  }
+
```
Comment (ajayesivan) [inline public/styles.css:434]: I'm not sure why you added these. In my view, it's not required. But if you think it's needed could you please explain why we need this?

File: public/styles.css [hunk 5]
```diff
@@ -75,6 +75,9 @@
   width: 130px;
   height: 80px;
 }
+.nav-menu{
+  z-index: 700;
+}
```
Comment (ajayesivan) [inline public/styles.css:80]: We already have a `.nav-menu` selector in this css file, please move the `z-index` there and remove this part.

File: public/styles.css [hunk 6]
```diff
@@ -75,6 +75,9 @@
...
 
 .menu {
   width: 18px;
```
Comment (ajayesivan) [inline public/styles.css:83]: Please remove the `width` and `height` properties from `.menu`, this is affecting the layout. Once we remove these properties we won't need the `margin-right` for `.nav-bar`.

**Review by ajayesivan — CHANGES_REQUESTED** [review]
@Adarshvarghese I have added view comments, please check and update the PR. If you have any questions please let me know.

File: public/styles.css (hunk 4 above)
Comment (Adarshvarghese) [reply public/styles.css:434]: When i gave these properties ,to me it seemed more closer to the design in figma.
There were not much  gap between the menu options in design.
Before giving  the properties in menu-option  , copyright@2023 leeway  was not visible in the bottom.
please let me know whether i have to make any changes.

Comment (ajayesivan) [reply public/styles.css:434]: Can you share a screenshot with and without this code?

Comment (Adarshvarghese) [reply public/styles.css:434]: before commenting
.menu-option {
    margin-bottom: 10px;
    padding-bottom: 25px;
  }
  .menu-nav{
    padding-top: 125px;
  }

before_commenting


After commenting out  :-


after_commenting

Comment (Adarshvarghese) [reply public/styles.css:434]: copyright@2023 leeway  is not visible when  padding is given to menu-nav class  without menu-option class .
I checked again and i think it looks fine without padding-bottom in menu-option class.
please have a look and let me know if there is any change required.

**Review by ajayesivan — APPROVED** [review]
LGTM! Great Work 🎉  @Adarshvarghese!

File: public/styles.css (hunk 4 above)
Comment (ajayesivan) [reply public/styles.css]: Your changes look good. Thanks for sharing the screenshot. I have approved the PR. You can squash & merge`

Comment (ajayesivan) [reply public/styles.css]: After merging this PR, please also close the duplicate PR(#84)

**Pull request merged**